statchatui/src/Compare/Compare.jsx

```diff
@@ -70,12 +78,12 @@ const Compare = ({ signedInUserData }) => {
     event.preventDefault();
 
     const gamertag = event.target.gamertag.value;
-    const platform = event.target.platform.value;
-    let gameName = event.target.gameName.value;
+    let platform = event.target.platform.value;
+    const gameName = event.target.gameName.value;
 
     //i did this because the api requires the apex legends xbox field to be X1 instead of Xbox, but for r6, it needs to stay xbox
     if (gameName === GAME_NAMES.APEX && platform === "Xbox") {
-      gameName = "X1";
+      platform = "X1";
     }
```

Comment:
I think it'd be good to move this into the fetchStats function, which I assume is being used in both Home and Compare.

Reply:
will do